database/wiiu/remove_friendship.go

```diff
@@ -1,21 +1,22 @@
 package database_wiiu
 
 import (
-	"github.com/PretendoNetwork/friends-secure/database"
-	"github.com/PretendoNetwork/friends-secure/globals"
+	"github.com/PretendoNetwork/friends/database"
 )
 
 // Remove a user's friend relationship
-func RemoveFriendship(user1_pid uint32, user2_pid uint32) {
+func RemoveFriendship(user1_pid uint32, user2_pid uint32) error {
```

Comment:
Again not your fault, but since we're here. This has a comment but it's in the wrong format